### ISSUE TITLE
Backward address mode 

### ISSUE THREAD

**Issue by Piradoxlanieve** [albertobsd/keyhunt#201]
Hola.

¿¿Seria posible implementar el modo hacia atrás en address mode?
Would it be possible to implement backwards mode in address mode

**Comment by albertobsd** [comment]
Yes that is some item in my to-do list but it is a lot of work, when i finish that part i will let you know.

Regards!
